config

```diff
@@ -0,0 +1,2 @@
+the:
```

Comment:
Was there any instruction on using config files? Or are we doing it for convenience?

Reply:
This is just one way of implementing the "the" parameters in the LUA code. The instructor said that we can use configurations.